source/ConfigMgrCBDsc.psd1

```diff
@@ -44,6 +44,7 @@
     DscResourcesToExport = @(
         'ClientSettings'
         'CMAccounts'
+        'SccmIniFile'
```

Comment:
Can't comment on the line but Tags could be updated as well

Reply:
Added